### DISCUSSION TITLE
co-browsing and screen share

### DISCUSSION THREAD

**Discussion by grenooye06** [LiveHelperChat/livehelperchat#2175] [General]
I have tried everything from the instructions provided but I just cant get either the co browsing or the Agora screen share to work...
When using the 'eye' on the chat tools it opens a 'Preview' popup. If using the screen share on the right-hand panel (I have configured all necessary from Agora and Settings), the process starts with the popupnand clicking on the blue eye in the top menu, the end-user accepts the 'Share screen' invitation but nothing is shared on the screen (blank screen). For further info, Agora enabled call and videos do not connect either (popups to start conversation do open but no connection takes place).
I have tried on another computer in case something was blocking on mine and exactly same result.
I have a separate co-browsing function on another section of my site which works perfectly.
Extra info a few days ago I first installe lhc from my hosting softaculous and the Agora connections worked fine but I could not get chatGPT bot to work so I decided to install directly from your repo and chatGPT worked like a charm but I totally lost screen sharing in the process.
Maybe there is anather older version of lch I could test, but I cant find anything but the latest version now (even on softaculous)?

**Comment by grenooye06** [comment]
I have found where I can find older versions and will be testing and reporting here if any progress

**Comment by grenooye06** [comment]
forgot to specify that my reported issues arise from version 4.52

**Comment by grenooye06** [comment]
No luck with any versions but I installed latest version on another hosting provider and it is running fine... So the problem must be with the way this hosting package is configured?

**Comment by remdex** [comment]
I'm not sure about the what hosting package you are talking. Next week you can write me directly on discord.

**Comment by grenooye06** [comment]
For further info, I cant get it to work on my NameCheap Stellar Plus Hosting php 8.2.26 but It works on my Hostgator shared hosting with php 8.2.18. Having said this voice connection not working... but I am only interested in screen share or co-browsing (still not working when clicking the chat 'eye' icon which opens an iframe stating 'Peview message'

**Comment by grenooye06** [comment]
Another problem which is more serious. When using screen-share and clicking on a link on the visitor's shared screen sends a cookie to the visitor which completely breaks the url (any page) in the browser with error 403. Access to the url still works on anoter browser and the only way to restore access is to delete cookies. I have managed to identify the culpret cookie (in my test location) as: 
cookie name: lhc_per, cookie value:  vid|ece0233ddb017ac272ee|hnh|1734760223|id|14|hash|YZVjgxg0bqPwDMLPQFwnt4IoDyexLyFW1II7xP8v
Again, this is only happening on my hosting package namecheap... tested without problem at hostgator
Would you know how to resolve this?

**Comment by grenooye06** [comment]
I resolved the problem with alterations to my .htaccess file:
![image](https://github.com/user-attachments/assets/7937d360-c866-4276-9c6e-12a311b21850)

**Comment by grenooye06** [comment]
Further investigations defined the problem to be due the length of the cookie required for screen-sharing.
So instead of disabling mod_securiy it is only necessary to change the configuration in .htaccess in order to allow longer cookies:
![image](https://github.com/user-attachments/assets/7d58e330-754a-4374-b7e0-2a2cc67f20f6)
